src/WebUI/locales/en.yml

```diff
@@ -502,13 +503,31 @@ character:
         notify:
           warning: A large shield cannot be used on horseback
       upgrade:
-        upgradesTitle: Item Upgrades
+        upgradesTitle: "Item upgrades"
+        tooltip:
+          title: "Upgrade item"
+          description: "You can upgrade your item for a heirloom point"
```

Comment:
Recommend change to -> "You can upgrade your item by spending one heirloom point"